Remove duplicated "ZRS" (#84966)
Azure/azure-content: articles/storage/files/storage-how-to-create-file-share.md

---

### Enable large files shares on an existing account

> Before you create an Azure file share on an existing account, you may want to enable it for large file shares if you haven't already. Standard storage accounts with LRS and ZRS can be upgraded to support large file shares. If you have a GRS, GZRS, RA-GRS, or RA-GZRS account, you will need to convert it to an LRS account before proceeding.

Reviewer: ```suggestion
Before you create an Azure file share on an existing account, you may want to enable it for large file shares if you haven't already. Standard storage accounts using either LRS or ZRS can be upgraded to support large file shares. If you have a GRS, GZRS, RA-GRS, or RA-GZRS account, you will need to convert it to an LRS account before proceeding.
```
